apps/condo/domains/organization/integrations/sbbol/sync/index.js

```diff
@@ -95,14 +100,22 @@ const sync = async ({ keystone, userInfo, tokenSet, reqId }) => {
         password: faker.internet.password(),
     }
 
+    const date = dayjs().format('YYYY-MM-DD')
     const user = await syncUser({ context, userInfo: userData })
     const organization = await syncOrganization({ context, user, userData, organizationInfo, dvSenderFields })
     const sbbolSecretStorage = getSbbolSecretStorage()
-    await sbbolSecretStorage.setOrganization(organization.id)
-    await syncTokens(tokenSet)
+    await sbbolSecretStorage.setOrganization(organization.id, user.id)
+    await syncTokens(tokenSet, user.id)
     await syncSubscriptions()
+    await syncSbbolTransactions.delay(date, user.id)
 
-    const organizationEmployee = await getOrganizationEmployee({ context, user, organization })
+    const { keystone: organizationContext } = await getSchemaCtx('Organization')
+    const organizationEmployee = await OrganizationEmployee.getOne(organizationContext, {
+        user: { id: user.id },
+        organization: {
+            id: organization.id,
+        },
+    })
     if (!organizationEmployee) {
         throw new Error('Failed to bind user to organization')
     }
```

Comment:
Maybe to move this check into `syncOrganization` and return employee from there?
Here you are getting an employee from database, but error message states about binding operation.